MLS.Agent.Tests/MLS.Agent.Tests.csproj

```diff
@@ -56,5 +56,12 @@
   <ItemGroup>
     <PackageReference Update="Microsoft.NET.Test.Sdk" Version="16.2.0" />
   </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Update="xunit.runner.visualstudio" Version="2.4.3">
```

Comment:
Don't add any references to this package; instead update it in the root `Directory.Build.targets`.

Or better yet, don't build with `$(DisableArcade) = 1` :smile:

Reply:
(this applies to all instances where the `xunit.runner.visualstudio` package was added)